sherpa/astro/background.py

```diff
@@ -17,13 +17,32 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+"""Support multiple background components for a PHA dataset.
+
+"""
+
 from sherpa.models.model import CompositeModel, ArithmeticModel
 from sherpa.utils.err import DataErr
 
 __all__ = ('BackgroundSumModel',)
 
 
 class BackgroundSumModel(CompositeModel, ArithmeticModel):
+    """Combine multiple background datasets.
+
+    Define the model expression to be applied to the source
+    region accounting for the backgroud models.
```

Comment:
I don't understand. Does the model define the expression?

Reply:
It's meant to be "model expression". It could have been "model", but I feel that is a bit anemic. However, I am not the best person to judge my words. 

There's no documentation on how this is meant to be used, so I'm relying on its's use in the following (sherpa/astro/ui/utils.py), where we see we need the models to be sent in as a dict (indexed by there "bkg_id" value).

```
    def _add_convolution_models(self, id, data, model, is_source):
        model = \
            sherpa.ui.utils.Session._add_convolution_models(self, id, data,
                                                            model, is_source)
        id = self._fix_id(id)
        if (isinstance(data, sherpa.astro.data.DataPHA) and is_source):
            if not data.subtracted:
                bkg_srcs = self._background_sources.get(self._fix_id(id), {})
                if len(bkg_srcs.keys()) != 0:
                    model = (model +
                             sherpa.astro.background.BackgroundSumModel
                             (data, bkg_srcs))

            pileup_model = self._pileup_models.get(self._fix_id(id))
            if pileup_model is not None:
                resp = \
                    sherpa.astro.instrument.PileupResponse1D(
                        data, pileup_model)
                model = resp(model)

            elif len(data._responses) > 1:
                resp = sherpa.astro.instrument.MultipleResponse1D(data)
                model = resp(model)

            else:
                resp = sherpa.astro.instrument.Response1D(data)
                model = resp(model)

        return model
```

Reply:
I've not got the energy to change this, so I'm going to punt on it and hope it gets improved whenever someone gets to write proper documentation for this class.